README.md

```diff
@@ -133,22 +135,23 @@ EscPos.addListener("bluetoothDeviceFound", (event: any) => {
 
 ## Design Tags
 
-| Tag      | Description                                   |
-| -------- | :-------------------------------------------- |
-| {B}      | Bold.                                         |
-| {U}      | Underline.                                    |
-| {H1}     | Font Size. 2x2 / char                         |
-| {H2}     | Font Size. 1x2 / char                         |
-| {H3}     | Font Size. 2x1 / char                         |
-| {LS:?}   | Linespace. M = 24LS, L = 30LS                 |
-| {C}      | Align text to center.                         |
-| {R}      | Align text to right.                          |
-| {RP:?:?} | Repeat text. Eg. {RP:5:a} will output "aaaaa" |
-| {QR[?]}  | Print QR code.                                |
-| {IMG[?]} | Print image from a path.                      |
-| {<>}     | Left-right text separation.                   |
-| {---}    | Create a "---" separator.                     |
-| {===}    | Create a "===" separator.                     |
+| Tag      | Description                                            |
+| -------- | :----------------------------------------------------- |
+| {B}      | Bold.                                                  |
+| {U}      | Underline.                                             |
+| {H1}     | Font Size. 2x2 / char                                  |
+| {H2}     | Font Size. 1x2 / char                                  |
+| {H3}     | Font Size. 2x1 / char                                  |
+| {LS:?}   | Linespace. M = 24LS, L = 30LS                          |
+| {C}      | Align text to center.                                  |
+| {R}      | Align text to right.                                   |
+| {RP:?:?} | Repeat text. Eg. {RP:5:a} will output "aaaaa"          |
+| {QR[?]}  | Print QR code.                                         |
+| {IMG[?]} | Print image from a path.                               |
+| {IWO:?}  | Add Image Width Offset. In practice reduces image size.|
```

Comment:
Better if we can combine this method with the method above ("IMG[?]"), we can make the second method optional, you can refer to `index.js` but I am not sure if that is the best practice.